product/examples/ProductCollectionOneatlas.json

```diff
@@ -0,0 +1,467 @@
+{
+  "conformsTo": [
+    "https://stat-api.example.com/v0.1.0/core",
+    "https://stat-api.example.com/v0.1.0/opportunities"
+  ],
+  "products": [
+    {
+      "type": "Product",
+      "stac_version": "1.0.0",
+      "stac_extensions": [],
+      "conformsTo": [
+        "https://geojson.org/schema/MultiPolygon.json",
+        "https://geojson.org/schema/Polygon.json"
+      ],
+      "id": "4f866cd3-d816-4c98-ace3-e6105623cf13",
+      "title": "PHR-tasking",
+      "description": "Very high resolution 50 cm optical satellite imagery, acquired worldwide with a daily revisit rate.",
+      "keywords": ["optical", "oneatlas", "tasking"],
+      "license": "proprietary",
+      "providers": [
+        {
+          "name": "oneatlas",
+          "description": "oneatlas description",
+          "roles": ["producer"],
+          "url": "https://intelligence.airbus.com/"
+        },
+        {
```

Comment:
i think you can remove these up42 providers?

Reply:
Give an example of why the providers are an array of objects.